coc-settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "diagnostic.virtualText": true,
+  "codeLens.enable": true,
+  "eslint.autoFixOnSave": true,
```

Comment:
I just tried adding this in locally and didn't have success. I did however get autoformatting working with these settings: 
```"coc.preferences.formatOnType": true,
  "coc.preferences.formatOnSaveFiletypes": [
    "javascript",
    "json",
    "elixir",
    "ruby",
    "css",
    "ts",
    "jsx",
    "tsx",
    "markdown"
  ], ```

Is there something else that needs to be updated for this to work?

Reply:
Good catch, eslint has its own flag for some reason. But you definitely need to add these for elixir and the other to auto format.